src/syft/lib/pandas/__init__.py

```diff
@@ -32,9 +34,13 @@ def create_ast(client: TypeAny = None) -> Globals:
     classes: TypeList[TypeTuple[str, str, TypeAny]] = [
         ("pandas.DataFrame", "pandas.DataFrame", pd.DataFrame),
         ("pandas.Series", "pandas.Series", pd.Series),
+        ("pandas.CategoricalDtype", "pandas.CategoricalDtype", pd.CategoricalDtype),
+        ("pandas.Categorical", "pandas.Categorical", pd.Categorical),
     ]
 
     methods: TypeList[TypeTuple[str, str]] = [
+        ("pandas.read_csv", "pandas.DataFrame"),
```

Comment:
Q: Is this safe to include as a "method" that could be called?

Reply:
@gmuraru I just checked the 'dev' branch and found it was removed after I started working on it. Thank you for pointing it out. It would be great if you could help me understand why it is unsafe to do so.

Reply:
Yes this is unsafe so we need to remove it again.

Reply:
I created a separate ticket where we will add these methods to a deny list which will check during any module loading and reject these as well, but for now this must be removed.